app/exceptions.py

```diff
@@ -6,3 +6,7 @@ def __init__(self, message):
 class NotificationTechnicalFailureException(Exception):
     def __init__(self, message):
         self.message = message
+
+
+class ValidationError(Exception):
```

Comment:
When I first saw this I thought it was the jsonschema ValidationError. Can we give it a different name?